src/pages/dashboard/.Map.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useMemo, useEffect } from 'react'
+
+import { json } from 'd3-fetch'
+import { nest } from 'd3-collection'
+import { sum } from 'd3-array'
+
+import _Map from './charts/_Map'
+import { path } from './charts/_Map'
+import StatesWithPopulation from './data/_state-populations'
+
+import { formatDate, formatNumber } from './.utils'
```

Comment:
What does the leading period signify in the filename here?

Reply:
i was unaware of this but it looks like either a `.` or `_` stop gatsby from trying to create a page.  Otherwise if you have `dashboard/utils.js` it tries to create a route at `/dashboard/utils/`

I was following the organizational suggestions set forth by @melbee altho I'm not sure i did it exactly as she suggested.  I'm going to have her review this as well to make sure we're on the same page.

Reply:
If I included a period before the file name that’s a typo on my end! 

import { formatDate, formatNumber } from './utils' 

Above is what I intended